medusa/tv/cache.py

```diff
@@ -51,7 +51,8 @@ def __init__(self, provider_id):
                     b'    url TEXT,'
                     b'    time NUMERIC,'
                     b'    quality NUMERIC,'
-                    b'    release_group TEXT)'.format(name=provider_id))
+                    b'    release_group TEXT'
```

Comment:
Missing `,`?

Reply:
yeah good catch. But strange enough it's not required.
I've added it anyway.